projects/systelab-components/src/lib/contextmenu/abstract-context-menu.component.ts

```diff
@@ -11,6 +11,10 @@ export abstract class AbstractContextMenuComponent<T> extends AbstractContextCom
 	@Output() public action = new EventEmitter();
 
 	public _contextMenuOptions: Array<T>;
+	protected previousShownMenu: Array<string> = new Array<string>();
+	protected previousMenuWidth: Array<number> = new Array<number>();
```

Comment:
protected previousMenuWidth: Array<number> = [];

Reply:
Thanks for your comment